packages/db/drizzle/0003_glamorous_living_mummy.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE `incidents_to_monitors` (
+	`monitor_id` integer NOT NULL,
+	`incident_id` integer NOT NULL,
+	PRIMARY KEY(`incident_id`, `monitor_id`),
+	FOREIGN KEY (`monitor_id`) REFERENCES `monitor`(`id`) ON UPDATE no action ON DELETE cascade,
+	FOREIGN KEY (`incident_id`) REFERENCES `incident`(`id`) ON UPDATE no action ON DELETE cascade
+);
```

Comment:
Currently getting the `SQLITE_CONSTRAINT: SQLite error: FOREIGN KEY constraint failed` when trying to insert entry.